### DISCUSSION TITLE
[Question] Is there eraser functionality?

### DISCUSSION THREAD

**Discussion by bybabek** [hm21/pro_image_editor#99] [Q&A] [answered]
### Placeholder

Is there eraser functionality? If not, is it going to be added soon?

**Comment by hm21** [comment]
Currently, I don't have any plans to add this functionality.
However, I can add it to the roadmap if you need it. When you delete a layer, is it enough when you hit a layer, and it removes the complete layer, or do you require that it just cut a part of it? 
For example, if we hit the middle of an arrow layer, should it remove the whole arrow or just cut the area in the middle? Removing the whole layer will be much faster to program.

**Comment by bybabek** [comment]
I would be thankful if you add it. I appreciate your work and it helps me in my task. I would prefer it like "paint" eraser. It should be able to erase some part of the layer. But other choice is fine by me as well. 
Thanks in advance.

**Comment by hm21** [comment, marked as answer]
Okay as a quick solution, I will first add the option where it removes the whole layer. Later I will extend it like in “paint” but that will take more time because I need to change a lot in the code for that.
